README.md

```diff
@@ -216,7 +218,37 @@ For a more in-depth understanding of the structure of these records, please expl
 In addition to the primary feature of interacting with Chainlink services, this plugin provides the ability to manage a local Chainlink node.
 The corresponding functionality is implemented in a special tasks module `sandbox`.
 This module implements methods for starting, restarting and stopping a Chainlink node, getting Chainlink node information,
-deploying and interacting with such contracts as [LinkToken](contracts%2FLinkToken.sol), [Operator](contracts%2FOperator.sol) and [ChainlinkDirectRequestConsumer](contracts%2FChainlinkDirectRequestConsumer.sol).
+deploying and interacting with such contracts as [LinkToken](contracts%2FLinkToken.sol), [Operator](contracts%2FOperator.sol), [FunctionsConsumer.sol](contracts%2FFunctionsConsumer.sol) and [ChainlinkDirectRequestConsumer](contracts%2FChainlinkDirectRequestConsumer.sol).
+
+### Configure and run Functions requests simulations
+
+This plugin allows you to run local Functions requests simulations.
+> **Note**  
+Install [Deno](https://deno.com/) and add it to PATH, run ```deno --version``` to verify installation. Instructions: [deno.land/#installation](deno.land/#installation).
+
+Before you run Functions requests simulations, you can configure it. To achieve this, parameters have been included in the Hardhat configuration `chainlink` group:
+`hardhat.config.ts`:
+```ts
+module.exports = {
```

Comment:
RIght now these are hard service limits.  Customizations to these are on-request and a case by case basis only.  We may not want to present these as being configurable for now?

Reply:
These parameters are extending the Hardhat config (https://hardhat.org/hardhat-runner/docs/advanced/building-plugins#extending-the-hardhat-config). And are supposed to be added to `hardhat.config.ts` of a user's Hardhat project (it is specified on the top of the code snippet). I will take a look and try to make it more clear.